src/components/Popups/index.tsx

```diff
@@ -33,40 +33,31 @@ const MobilePopupInner = styled.div`
   }
 `
 
-const StopOverflowQuery = `@media screen and (min-width: ${MEDIA_WIDTHS.deprecated_upToMedium + 1}px) and (max-width: ${
-  MEDIA_WIDTHS.deprecated_upToMedium + 500
-}px)`
-
-const FixedPopupColumn = styled(AutoColumn)<{ extraPadding: boolean; xlPadding: boolean }>`
+const FixedPopupColumn = styled(AutoColumn)<{
+  drawerOpen: boolean
+}>`
   position: fixed;
-  top: ${({ extraPadding }) => (extraPadding ? '72px' : '64px')};
+  top: ${({ drawerOpen }) => `${64 + (drawerOpen ? -50 : 0)}px`};
```

Comment:
Can you leave a comment explaining what this is doing?
Why is it better to move this up vertically rather than just use z-index to put it over the drawer?

Reply:
hard to add a comment in the code there due to jsx tag, but it does use z-index to go over drawer, it just also shift itself up or down vertically so that when drawer is closed it goes below the header, whereas when its open, it shifts up. visually looks much better.